maxim/logger/bedrock/async_client.py

```
@@ -14,11 +14,24 @@
 class MaximBedrockAsyncClient:
+    """Maxim Bedrock async client wrapper.
+
+    This class provides a wrapper around the Bedrock async client to integrate
+    with Maxim's logging and monitoring capabilities. It allows tracking
+    and logging of Bedrock API interactions through the Maxim platform.
+    """
+
     def __init__(
         self,
         logger: Logger,
         client: BaseClient,        
```

Flake8 (7.2.0)

[warning] trailing whitespace

(W291)